gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-bin.zip
```

Comment:
What is the reason for this change?

Reply:
Gradle 6.6/6.7 cause errors with kotlin on some specific java versions: gradle/gradle#14155.

Either 6.5.1 or 6.8.3 works for me.

Reply:
I see. It might be better to stick to the more recent version if that is the case then.